File: PROGRAMMING/iiitv/S E M - 1/Programming/Assignments/Assignment 5/code3.c
#include <stdio.h>

int main() {
    int num;
    int count = 0, digits[10];

    printf("Enter an integer (more than 4 digits): ");
    scanf("%d", &num);

    int temp = num;
    while (temp > 0) {
        digits[count] = temp % 10;
        temp = temp / 10;
        count++;
    }

    int output = digits[count-1] * 1000 + digits[count-2] * 100 + digits[count-3] * 10 + digits[count-4];
    printf("%10d\n", output);

    return 0;
}

File: COLLEGE/iiitv/S E M - 2/Java/Assignment 4/code3.java
public class code3 {
    public static void main(String[] args) {
        Temp t1 = new Temp();

        System.out.println(t1.num);
        t1.num = 20;

        Temp t2 = new Temp();
        System.out.println(t2.num);
    }
}

class Temp{
    public static int num = 10;
}
